mqtt-service-examples/mqtt-service-hello/src/main/java/c8y/example/mqtt/service/microservice/StandaloneRunner.java

```diff
@@ -0,0 +1,66 @@
+package c8y.example.mqtt.service.microservice;
+
+import com.cumulocity.model.authentication.CumulocityCredentialsFactory;
+import com.cumulocity.mqtt.service.client.MqttClient;
+import com.cumulocity.mqtt.service.client.MqttConfig;
+import com.cumulocity.mqtt.service.client.MqttMessageListener;
+import com.cumulocity.mqtt.service.client.MqttSubscriber;
+import com.cumulocity.mqtt.service.client.model.MqttMessage;
+import com.cumulocity.sdk.client.Platform;
+import com.cumulocity.sdk.client.PlatformImpl;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.concurrent.CountDownLatch;
+
+import static com.google.common.base.Preconditions.checkArgument;
+
+@Slf4j
+public class StandaloneRunner {
```

Comment:
Suggestion: this class looks a bit out of place to me, maybe we should move it to separate module? So we would have:
* *mqtt-service-hello* -- which would be only this class (+ pom.xml and logback.xml)
* *mqtt-service-hello-microservice* -- so basically this module with name signifying that it is the same, but using microservice SDK

Reply:
For now I just move it to the test package and renamed to `LocalIntegrationTest`